### PR TITLE
Update custom status, getting ready for the website update.

### PR DESCRIPTION
**Pull request by eggsy**
Not for testing purposes anymore 😎 Check out the new page [here](https://eggsy-tailwind.netlify.app/projects/premid/custom-status) and see [here](https://eggsy.xyz/projects/premid/custom-status) that it still working on the old one as well.

image

### CODE REVIEW

File: websites/C/Custom Status/dist/metadata.json [hunk 1]
```diff
@@ -8,12 +9,12 @@
...
   },
-  "url": "eggsy.xyz",
-  "regExp": "eggsy[.]xyz[/]projects[/]premid[/]custom-status",
-  "version": "1.2.5",
-  "logo": "https://premid.is-inside.me/PsLr519L.PNG",
-  "thumbnail": "https://premid.is-inside.me/rWy3ggow.png",
-  "color": "#212121",
+  "url": ["eggsy.xyz", "eggsy-tailwind.netlify.app"],
+  "regExp": "[eggsy[.]xyz[/]projects[/]premid[/]custom-status|eggsy-tailwind[.]netlify[.]app[/]projects[/]premid[/]custom-status",
+  "version": "1.5.0",
```
Comment (Bas950) [inline websites/C/Custom Status/dist/metadata.json:14]: dont think you mean to put the `[` at the begin of the regex
and 1.2.5 to 1.5.0 at once? just go to 1.3.0

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/C/Custom Status/dist/metadata.json (hunk 1 above)
Comment (eggsy) [reply websites/C/Custom Status/dist/metadata.json]: but it's a big update :((

**Review by Bas950 — APPROVED** [review]

File: websites/C/Custom Status/dist/metadata.json (hunk 1 above)
Comment (ririxidev) [reply websites/C/Custom Status/dist/metadata.json]: If you change only metadata it should be 1.2.6 xD

Comment (eggsy) [reply websites/C/Custom Status/dist/metadata.json]: :p

**Review by ririxidev — CHANGES_REQUESTED** [review]
upload proof of working presence :)

**Comment by eggsy** [comment]
> upload proof of working presence :)

I am still working on and code hasn't changed, what changed is the website, it works with the old website and if it doesn't work with the new website, I'll update the website, not the presence, so I guess there's no need to upload any proof to this, just see this as a metadata change like you said

**Comment by ririxidev** [comment]
I wanna say, if you are doing this for testing purposes it shouldn't be even merged since you can check it by local presence.

**Comment by ririxidev** [comment]
Closing then. Test it by local presence :)

**Review by ririxidev — DISMISSED** [review]

**Review by Bas950 — CHANGES_REQUESTED** [review]
Nothing changed in your last commit? You just un-prettified the tags

**Comment by ririxidev** [comment]
Only thing that I wanted, I wanted screenshot of working presence for his new site. He was saying it is for testing purposes. Now he has it working so yeah

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**